### PR TITLE
Pal 1030 deploy to ref

### PR DESCRIPTION
**Pull request by pd104923**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@Library('jenkinsfile-lib')_
+@Library('jenkinsfile-lib@feature/PAL-1030-deploy-to-ref')_
```
Comment (pd104923) [inline Jenkinsfile:16]: This needs to be reverted.

File: licenses/mit_license.html [hunk 2]
```diff
@@ -18,7 +18,7 @@
 <link type="text/css" rel="stylesheet" href="https://opensource.org/files/css/css_7rptTjxTlnnwcQOT_3B6dmWJfzo0pnNKsCvuv3HArrk.css" media="all" />
 
 <!--[if (lt IE 9)]>
-<link type="text/css" rel="stylesheet" href="https://opensource.org/sites/all/themes/bootstrap-business/css/ie8.css?qqdhfs" media="all" />
+<link type="text/css" rel="stylesheet" href="https://opensource.org/sites/all/themes/bootstrap-business/css/ie8.css?qsj6ct" media="all" />
```
Comment (pd104923) [inline licenses/mit_license.html]: This license will be reverted

**Review by m78233 — DISMISSED** [review]
I think that is right

File: Jenkinsfile (hunk 1 above)
Comment (developer01189998819991197253) [inline Jenkinsfile]:
```suggestion
@Library('jenkinsfile-lib')_
```

File: templates/kafka-proxy-svc.yaml [hunk 3]
```diff
@@ -31,8 +31,8 @@ spec:
   type: ExternalName
   # Specify the FQDN of the service to use. Pod selectors can't select outside of namespace
   # Docs suggest FQDN can be shortened inside current namespace, and inside current cluster, but this doesn't work in AWS
-  {{- if eq .Values.global.hosting "aws" }}
+  {{- if .Values.global.kafka.install }}
```
Comment (developer01189998819991197253) [inline templates/kafka-proxy-svc.yaml:34]: wouldnt it makes sense to do an `if you deployed kafka and redis, and the hosting is aws`? 
Doesnt this affect the local deploy?

**Review by developer01189998819991197253 — CHANGES_REQUESTED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by dev930018 — APPROVED** [review]

**Pull request merged**